python/tests/unit/test_irratelimit.py

```diff
@@ -80,31 +64,9 @@ def _get_ratelimit_default_conf_v2():
     }
 
 
-@pytest.mark.compilertest
-def test_irratelimit_defaultsv3():
-    default_config = _get_ratelimit_default_conf_v3()
-
-    # Test all defaults
-    yaml = """
-apiVersion: getambassador.io/v3alpha1
-kind: RateLimitService
-metadata:
-  name: myrls
-  namespace: default
-spec:
-  service: {}
-""".format(SERVICE_NAME)
-    econf = _get_envoy_config(yaml, version='V3')
-    conf = _get_rl_config(econf.as_dict())
-
-    assert conf
-
-    assert conf.get('typed_config') == default_config
-
-
 @pytest.mark.compilertest
 def test_irratelimit_defaults():
-    default_config = _get_ratelimit_default_conf_v2()
+    default_config = _get_ratelimit_default_conf_v3()
```

Comment:
Doesnt this need to be `_get_ratelimit_default_conf()` I don't see a function for `_v3` anywhere in this file (nor do we need to specify v3/v2 anymore) so I'm curious how this is passing.